document_definition/README.rst

```diff
@@ -0,0 +1,54 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+
+===================
+document_definition
+===================
+
+Installs a new  document_definition model for dictionary-style word archive and
+definition. Searchable in the Knowledge menu.
+
+Known issues / Roadmap
+======================
+
+* A search function per term and per definition, obviously.
+* Dictionary Style Alphabetical grouping view
```

Comment:
Something I am also missing (but maybe I have overlooked it, or am just not understanding completely) is a view or field that allows the user to link the terms to a knowledge base document?

Reply:
in the view "Definitions" when expanding "advanced search" you will see an option "alphabetical" where the terms are grouped by their first letter, like a dictionary.